utils/database_relations.py

```
@@ -0,0 +1,317 @@
+            
+            await db.commit()
+            
+            return {
+                "issue_id": issue_id,
+                "session_id": session_id,
+                "status": "created"
+            }
+            
+        except Exception as e:
+            await db.rollback()
+            raise Exception(f"ChatbotIssue와 Session 생성 실패: {str(e)}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)